### ISSUE TITLE
How to use bot

### ISSUE THREAD

**Issue by MaZyGer** [Rusketh/ExpAdv2#268]
Hello I finally got bot working but I do not get how to loop behavoir.

Example bot shall follow me but if I use moveto he runs to getPos of owner but stops there even I am as owner moving away. I thought behavoir is coroutine which has something like endless loop.

```
        event behavior() {
            moveTo(owner().pos())
            print("Ended. No loop :(" )
        }   
```

**Comment by Henningstone** [comment]
so then, why don't you just put it into a loop?

**Comment by MaZyGer** [comment]
You are right. I thought it will give stack overflow error or something with tick. I thought behavoir itself is a loop :D

**Comment by VINTproYKT** [comment]
Hm, on more servers bot is not moving, I can change the activities all the while, but it doesn't move.

**Comment by Rusketh** [comment]
@VINTproYKT  the server needs a nav mesh generated or uploaded for nextbot to work.

**Comment by VINTproYKT** [comment]
@Rusketh Got it. Anyways, it will be awesome to move bots in naturally way (without force of `setPos`) even without nav mesh.
Sometimes I think  that I have very ugly nav mesh on `gm_flatgrass`, cause bots on this map are using stupid and ununified ways.

**Comment by Rusketh** [comment]
Already do-able.
